README.md

```diff
@@ -75,7 +86,12 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+
+var pizza = {
+  temp : "70",
+  bake : temp.replace("70","300"),  
+}
+
 ```
```

Comment:
`bake` should be a method (i.e., a function within the `pizza` object). Also, when referencing `temp` inside of `bake`, it needs to done using `this.temp`.